tests/Auth/bot-authentication/Bots/EchoBot.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading;
```

Comment:
Do test projects need license headers? We don't use test headers in the JS Tests, but do we do that here?

Reply:
absolutely